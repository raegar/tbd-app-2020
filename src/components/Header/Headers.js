import React from 'react';
import UCPLogo  from '../../images/UCPLogo.png';
import './Headers.css';
import {isMobile} from 'react-device-detect';

class Headers extends React.Component {
    render = () => {
    	const className = isMobile ? "MobileUCPLogoBlue" : "UCPLogoBlue";

    	return (
    		<header>   
    			<a href="/tbd-app-2020"><img 
    				data-testid="ucpLogoHeader" 
    				src={UCPLogo} 
    				className={className} 
    				alt="UCP official logo" 
    			/></a>  
    		</header>
    	);
    }
}

export default Headers;



