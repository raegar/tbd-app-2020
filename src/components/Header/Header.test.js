import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import Headers from './Headers';

//cleans up 
afterEach(cleanup);

it("Renders Header", () => {
    const { asFragment } = render(<Headers/>);
    expect(asFragment()).toMatchSnapshot();
});
