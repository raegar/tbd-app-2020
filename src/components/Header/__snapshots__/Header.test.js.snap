// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Header 1`] = `
<DocumentFragment>
  <div>
    <header
      class="Header"
    >
      <a
        href="/tbd-app-2020"
      >
        <img
          alt="UCP official logo"
          class="UCPLogoBlue"
          data-testid="ucpLogoHeader"
          src="Logo_White.png"
        />
      </a>
    </header>
    <div
      class="Line"
    />
  </div>
</DocumentFragment>
`;
